at_cookbook/Complete_Version/.flutter-plugins-dependencies

```diff
@@ -1 +1 @@
name":"webview_flutter","path":"C:\\\\src\\\\flutter_windows_1.22.4-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\webview_flutter-2.0.9\\\\","dependencies":[]}],"macos":[{"name":"path_provider_macos","path":"C:\\\\src\\\\flutter_windows_1.22.4-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\path_provider_macos-2.0.0\\\\","dependencies":[]},{"name":"url_launcher_macos","path":"C:\\\\src\\\\flutter_windows_1.22.4-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\url_launcher_macos-2.0.0\\\\","dependencies":[]}],"linux":[{"name":"path_provider_linux","path":"C:\\\\src\\\\flutter_windows_1.22.4-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\path_provider_linux-2.0.0\\\\","dependencies":[]},{"name":"url_launcher_linux","path":"C:\\\\src\\\\flutter_windows_1.22.4-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\url_launcher_linux-2.0.0\\\\","dependencies":[]}],"windows":[{"name":"path_provider_windows","path":"C:\\\\src\\\\flutter_windows_1.22.4-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\path_provider_windows-2.0.1\\\\","dependencies":[]},{"name":"url_launcher_windows","path":"C:\\\\src\\\\flutter_windows_1.22.4-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\url_launcher_windows-2.0.0\\\\","dependencies":[]}],"web":[{"name":"file_picker","path":"C:\\\\src\\\\flutter_windows_1.22.4-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\file_picker-3.0.3\\\\","dependencies":[]},{"name":"url_launcher_web","path":"C:\\\\src\\\\flutter_windows_1.22.4-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\url_launcher_web-2.0.1\\\\","dependencies":[]}]},"dependencyGraph":[{"name":"at_backupkey_flutter","dependencies":["share","permission_handler","path_provider"]},{"name":"at_onboarding_flutter","dependencies":["file_picker","flutter_qr_reader","url_launcher","permission_handler","path_provider","webview_flutter","at_backupkey_flutter"]},{"name":"file_picker","dependencies":["flutter_plugin_android_lifecycle"]},{"name":"flutter_keychain","dependencies":[]},{"name":"flutter_plugin_android_lifecycle","dependencies":[]},{"name":"flutter_qr_reader","dependencies":[]},{"name":"path_provider","dependencies":["path_provider_macos","path_provider_linux","path_provider_windows"]},{"name":"path_provider_linux","dependencies":[]},{"name":"path_provider_macos","dependencies":[]},{"name":"path_provider_windows","dependencies":[]},{"name":"permission_handler","dependencies":[]},{"name":"share","dependencies":[]},{"name":"url_launcher","dependencies":["url_launcher_linux","url_launcher_macos","url_launcher_web","url_launcher_windows"]},{"name":"url_launcher_linux","dependencies":[]},{"name":"url_launcher_macos","dependencies":[]},{"name":"url_launcher_web","dependencies":[]},{"name":"url_launcher_windows","dependencies":[]},{"name":"webview_flutter","dependencies":[]}],"date_created":"2021-07-01 20:26:03.885666","version":"2.2.2"}
```

Comment:
This should be in .gitignore (and not here)

Reply:
Noted